test-requirements.txt

```diff
@@ -8,6 +8,7 @@ docutils>=0.11 # OSI-Approved Open Source, Public Domain
 fixtures>=3.0.0 # Apache-2.0/BSD
 metalsmith
 mock>=3.0.0 # BSD
+munch>=3.0.0 # MIT
```

Comment:
Why is this needed?

Reply:
Was getting the error - ModuleNotFoundError: No module named 'munch' .
The specific failure occurred while trying to import the test module esiclient.tests.unit.v1.mdc.test_mdc_node_baremetal.

It seems that the test module test_mdc_node_baremetal.py has a dependency on the munch module, but it is not installed in our test environment.
In earlier PR's, they had different python versions and packages, the packages didn't depend on the munch

Reply:
Ah, I see - good catch!